docs/adr/0009-remote-types-interfaces-and-proc-macros.md

```diff
@@ -160,14 +160,14 @@ pub enum LogLevel {
 
 /// Define a custom type conversion from `log::Level` to the above type.
 uniffi::custom_type!(log::Level, LogLevel, {
-  from_custom: |l| match l {
+  into_existing: |l| match l {
```

Comment:
The word `existing` doesn't sound right to me. Everything is existing, really. They way I see it, is that we have `builtin` types which come from Rust, we have `custom` types which we create, and we have `external` types from other crates.